packages/contractkit/src/wrappers/Validators.ts

```diff
@@ -244,7 +245,14 @@ export class ValidatorsWrapper extends BaseWrapper<Validators> {
   /** Get list of registered validators */
   async getRegisteredValidators(): Promise<Validator[]> {
     const vgAddresses = await this.getRegisteredValidatorsAddresses()
-    return Promise.all(vgAddresses.map((addr) => this.getValidator(addr)))
+    let validators = await Promise.all(vgAddresses.map((addr) => this.getValidator(addr)))
+    const accounts = await this.kit.contracts.getAccounts()
+    return Promise.all(
+      validators.map(async function(v) {
+        v.name = (await accounts.getName(v.address)) || ''
```

Comment:
How about we set this in `getValidator` instead? And please update any other CLI commands that show validators to print the name if they don't already

Reply:
Looks like validator:show and election:current already show name

Reply:
Awesome! Will be nice to have those populated as well "for free" 😄